packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -128,7 +128,7 @@ class GoogleSignIn {
       );
     }
     await _initialization;
-    Map<String, dynamic> response = await _channel.invokeMethod(method);
+    Map<String, dynamic> response = await _channel.invokeMethod(method,null);
```

Comment:
This seems strange. `null` is the default value if nothing is passed, I believe. This shouldn't change anything...